TeamCode/src/main/java/org/firstinspires/ftc/teamcode/controls/def/button/ButtonCtlDef.java

```diff
@@ -0,0 +1,49 @@
+package org.firstinspires.ftc.teamcode.controls.def.button;
+
+import com.arcrobotics.ftclib.gamepad.GamepadEx;
+import com.arcrobotics.ftclib.gamepad.GamepadKeys;
+
+import org.firstinspires.ftc.teamcode.controls.def.ControlDefinition;
+
+import java.util.function.Consumer;
+
+public class ButtonCtlDef implements ControlDefinition {
```

Comment:
It feels like the CtlDef shorthand for ControlDefinition isn't necessary, but who cares really

Reply:
nu as bbd rasyt new ButtonControlDefinition kai controlsus reikes definint, pagal mane BtnCtlDef vps geriau butu ir less clutter pagal mane, nors ir more consifusin

Reply:
fair enough 🛐